healthcheck.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+set -e
+
+head () {
+    polkadot-js-api --ws ws://127.0.0.1:9944 query.system.number 2>/dev/null |\
```

Comment:
I don't think this endpoint performs the check for synchronization. The health check should respond with failure in case node is in syncing process at the moment. Please, read [the doc](https://www.notion.so/cere/Setup-health-check-for-a-blockchain-node-56de685341ff4ee48b467cf8b68123d8) I sent you previously. We should mimic the /[health/readiness](https://github.com/paritytech/polkadot-sdk/pull/4802/files#diff-6bc65bd838e6f221a64a7eb49bb3f02c743541bdfd9a2c4960bd5d28ab34a85eR110) endpoint in our health check.

Reply:
Sure. Let me address this.